apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -63,6 +63,7 @@ const initialState = {
   studentData: {},
   editingData: {},
   sectionId: null,
+  showSharingColumn: false,
```

Comment:
I know I just snuck this comment in, but it'd be great to add an explanatory comment above for this new feild.